src/Plan/Contracts/IntervalGeneratorContract.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+
+namespace Laravel\Cashier\Plan\Contracts;
+
+use Laravel\Cashier\Subscription;
+
+interface IntervalGeneratorContract
+{
+    /**
+     * @param \Laravel\Cashier\Subscription $subscription
+     *
+     * @return \Carbon\Carbon
+     */
+    public function getEndOfTheNextSubscriptionCycle(Subscription  $subscription = null);
```

Comment:
Remove double space before `$subscription`.
Rename to `getEndOfNextSubscriptionCycle`